src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java

```diff
@@ -203,7 +217,16 @@ private void transmitBatch(List<Transmission> transmissions) {
         });
     }
 
-    // email addresses is a string of email addresses with a space in between
+    /**
+     * Send email about the transmission to specified email addresses.
+     *
+     * @param transmission       the transmission we are concerned with
+     * @param confirmationNumber the confirmation number of the application
+     * @param emailAddresses     the email addresses to send it to - these should be a string of email addresses with a space in
+     *                           between addresses
```

Comment:
non-blocking: Consider giving this API parameter a more semantically-accurate type (`List<String>`)

Reply:
Noting this for future work.
https://www.pivotaltracker.com/story/show/187296734